src/IIIF/IIIF.Tests/ImageApi/ImageRequestXTests.cs

```diff
@@ -221,4 +221,86 @@ public void Parse_CorrectPercentageScaled()
         // Assert
         result.Should().BeEquivalentTo(expected);
     }
+    
+    [Theory]
+    [InlineData("my-asset")]
+    [InlineData("my-asset/")]
+    public void Parse_IsBase(string path)
+    {
+        // Arrange and Act
+        const string prefix = "iiif-img/27/1/";
+        var result = ImageRequest.Parse($"{prefix}{path}", prefix);
+        
+        // Assert
+        result.IsBase.Should().BeTrue();
+    }
+    
+    [Fact]
+    public void Parse_InfoJson()
+    {
+        // Arrange and Act
+        const string prefix = "iiif-img/27/1/";
+        var action = () => ImageRequest.Parse($"{prefix}my-asset/info.json", prefix);
+        
+        // Assert
+        action.Should().NotThrow<ArgumentException>();
+    }
+    
+    [Fact]
+    public void Parse_Fails_WhenInfoHasInvalidExtension()
+    {
+        // Arrange and Act
+        const string prefix = "iiif-img/27/1/";
+        var action = () => ImageRequest.Parse($"{prefix}my-asset/info.jsonll", prefix);
+        
+        // Assert
+        action.Should().ThrowExactly<ArgumentException>();
+    }
+
+    [Theory]
+    [InlineData("iiif-img/27/1/")]
+    [InlineData("/iiif-img/27/1/")]
+    [InlineData("/iiif-img/27/1")]
+    [InlineData("iiif-img/27/1")]
+    public void Parse_Validate_Succeeds(string prefix)

```

Comment:
More descriptive name, this is testing whether trailing/leading slashes can be handled for prefix